### PR TITLE
Added a guide on how to configure to use SSL

### PR DESCRIPTION
**Pull request by eanker**
As discussed in IRL, the rest of the changes was accidentally pushed on master, because no master was created when pushing to this code.

Testing can be done by going to for example: `https://localhost:8443/message/whatever`

To ignore the jetty-ssl-context.xml changes use: 
`git update-index --assume-unchanged src/main/resources/jetty-ssl-context.xml`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# backend
+
+Running the backend with ssl:
+* 1) Go to `src/main/resources` in the terminal
+* 2) exacute: `keytool -genkey -alias sitename -keyalg RSA -keystore keystore.jks -keysize 2048`
```
Comment (TimvdLippe) [inline README.md]: *Execute

Comment (TimvdLippe) [inline README.md]: `sitename` should be replaced with `jetty`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+with
+First and Last name: `localhost`,
+Fill the rest in as you wish
+and save your password somewhere
+* 2) `keytool -delete -alias sitename -keystore keystore.jks`
```
Comment (TimvdLippe) [inline README.md]: `sitename` should be replaced with `jetty`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+* 3) `keytool -list -keystore keystore.jks`
+verify that the keytool has no entries
+* 4) Put all the certificates temporaily in your resource folder or use the path to it when using `server.*`.
```
Comment (TimvdLippe) [inline README.md]: *temporarily

Comment (TimvdLippe) [inline README.md]: "in the folder `src/main/resources/`" is a bit more explicit. Also I do not understand what the second part of the sentence (about the `server.*`) means. Could you elaborate?

**Comment by TimvdLippe** [comment]
If you use start with `1.` and then for every step start the sentence with `*` the numbers will autoincrement. See the example list in https://raw.githubusercontent.com/preview-code/rite-evaluation/master/README.md

File: README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+When asked for a password use the password from step 2.
+* 5) `keytool -keystore keystore -import -alias jetty -file server.crt -trustcacerts`
+* 6) `openssl pkcs12 -inkey server.key -in server.crt -export -out server.pkcs12`
+* 7) `keytool -importkeystore -srckeystore server.pkcs12 -srcstoretype PKCS12 -destkeystore keystore.jks`
+* 8) Put your password in `src/main/resources/jetty-ssl-context.xml`
+* 9) Run `mvn jetty:run`
+* 10) Go to `https://localhost:8443/message/whatever`
```
Comment (TimvdLippe) [inline README.md]: This will break when we have an actual correct server running. Probably we can omit this step.

File: README.md (hunk 1 above)
Comment (eanker) [inline README.md]: No, because this certificate is deleted in the next step. So the name is non-important.

File: README.md (hunk 2 above)
Comment (eanker) [inline README.md]: Only if you have changed it in step 1.

File: README.md (hunk 3 above)
Comment (eanker) [inline README.md]: image

**Comment by eanker** [comment]
I updated the readme to be more clear, however I have not changed everything.
See, my comments on your outdated diffs.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+with 
+first and Last name: `localhost`,
+fill the rest in as you wish
+and save your password somewhere
+*  `keytool -delete -alias sitename -keystore keystore.jks`
+*  `keytool -list -keystore keystore.jks`
+verify that the keytool has no entries
+*  Put all the certificates temporarily in your resource folder or use the path to the certificate files, when the names are used in the next steps. The files are named `server.crt` and `server.key`.
+When asked for a password use the password from step 2.
+*  `keytool -keystore keystore -import -alias jetty -file server.crt -trustcacerts`
```
Comment (TimvdLippe) [inline README.md]: `-keystore keystore.jks`

**Comment by TimvdLippe** [comment]
Guide works! Was missing the `firebase-auth.json` file though, so that should be added in the Installations guidelines.

**Comment by TimvdLippe** [comment]
LGTM!

**Pull request merged**